setup.py

```diff
@@ -0,0 +1,48 @@
+import setuptools
```

Comment:
Could you add a brief note to the main README, to the documentation, and here, explaining when this file needs to be changed to maintain working pip support? Our development and test workflow doesn't currently use PIP, so this will be easy to break.

Reply:
How about we move the CircleCI setup to _only_ use PIP? I assume that the differences between PIP/non-PIP setups will generally be small, and that most developers will be testing in a non-PIP setup locally.

Reply:
Works for me. I'll update that.